lib/assembly/object_file.rb

```diff
@@ -88,13 +88,13 @@ def exif
     # @return [String] computed md5 checksum
     def md5
       check_for_file unless @md5
-      @md5 ||= Digest::MD5.file(path).hexdigest
+      @md5 ||= Digest(:MD5).file(path).hexdigest
```

Comment:
oh boy.  I'll bet this is in moab-versioning ....

Reply:
Fortunately only in one spot: https://github.com/sul-dlss/moab-versioning/blob/main/lib/moab/file_signature.rb#L64-L66 I'll patch it. 🍻